_calls/fields-search.md

```diff
@@ -0,0 +1,532 @@
+---
+layout: call
```

Comment:
Use a [default value](https://github.com/sgmap/beta.gouv.fr/blob/gh-pages/_config.yml#L20-L27) for the collection to avoid repeating this.